cmd/root.go

```diff
@@ -47,11 +49,18 @@ var root = &cobra.Command{
 			return errors.New("could not get current user")
 		}
 
+		m, metaExists := meta.New().ParseHeader(slides[0])
+		if metaExists {
```

Comment:
```suggestion
		m, exists := meta.New().ParseHeader(slides[0])
		// If the user specifies a custom configuration options
		// skip the first "slide" since this is all configuration
		if exists {
```